components/Button/buttonPrimary.style.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "@emotion/styled";
+import primaryTheme from "../../styles/theme/theme";
+
+const Button = styled('a')({
+  backgroundColor: primaryTheme.colors.secondary,
+  color: primaryTheme.colors.primary,
+  padding: '10px 70px',
```

Comment:
We should use rems instead of px.

Also per my comment on the font's PR, I would suggest creating a folder for example global-styles and add there files for global use. For example, colors file, breakpoints, fonts, space etc.

You can create a file named space having the below values (we can add/change more): 

`const space = [
  '0',
  '0.4rem',
  '0.8rem',
  '1.2rem',
  '1.6rem',
  '2.4rem',
  '3.2rem',
  '4.8rem',
  '7.2rem',
]

space.none = space[0] // 0px
space.xsmall = space[1] // 4px
space.small = space[2] // 8px
space.medium = space[3] // 12px
space.base = space[4] // 16px
space.large = space[5] // 24px
space.xlarge = space[6] // 32px
space.xxlarge = space[7] // 48px
space.xxxlarge = space[8] // 72px

export default space`

Reply:
@dimitrios-raptis agreed, but this would better be done for CP-22 I think. Happy to do it after these two PRs are resolved.